noxfile.py

```diff
@@ -66,15 +65,19 @@ def conda_tests(session):
     session.run("pytest", *tests)
 
 
-@nox.session
+@nox.session(python=["3.6", "3.7", "3.8", "3.9", "3.10"])
 def cover(session):
     """Coverage analysis."""
     if ON_WINDOWS_CI:
         return
 
+    # 3.10 produces different coverage results for some reason
+    # see https://github.com/theacodes/nox/issues/478
+    fail_under = 99 if is_python_version(session, "3.10.0") else 100
```

Comment:
Instead of parametrising this session, could we just check `sys.version_info` here?

Reply:
Yeah I like this idea, worked locally for me